setup.py

```diff
@@ -35,24 +35,26 @@
 
 requirements = [
     'Flask>=0.10.1',
-    'invenio-base>=0.1.0',
-    'invenio-celery>=0.1.0',
-    'invenio-ext>=0.2.1',
-    'invenio-records>=0.2.0',
-    'invenio-upgrader>=0.1.0',
-    'invenio-utils>=0.1.1',
-    'invenio-workflows>=0.1.0',
+    'invenio-base>=0.3.1',
+    'invenio-celery>=0.1.1',
+    'invenio-ext>=0.3.1',
+    'invenio-records>=0.3.4.post1',
+    'invenio-upgrader>=0.2.0',
+    'invenio-utils>=0.2.0',
+    'invenio-workflows>=0.1.2',
```

Comment:
Don't you want to force max version as well for `invenio-*` packages? So that you don't have the 1.0 version installed until it's compatible.